geometry_msgs/Wrench[] wrenches
string[] parent_names
string[] child_names